support/Procfile

```diff
@@ -1,7 +1,6 @@
-redis: redis-server --dir tmp
 web: support/builder_web.sh
```

Comment:
`support/builder_web.sh` doesn't have an execute bit so this procfile doesn't start up cleanly. Easy fix!